### ISSUE TITLE
Unable to import

### ISSUE THREAD

**Issue by bitworm** [spatialaudio/python-sounddevice#115]
I tried to install this per instructions on the webpage and get this error:
 ```
 2632     """Initialize PortAudio."""
   2633     global _initialized
-> 2634     _check(_lib.Pa_Initialize(), 'Error initializing PortAudio')
   2635     _initialized += 1
   2636

error: symbol 'Pa_Initialize' not found in library '<None>': dlsym(RTLD_DEFAULT, Pa_Initialize): symbol not found
```
All dependencies were satisfied (according to webpage instructions).  Any thoughts?

Other info: macos, python 3.5.

**Comment by mgeier** [comment]
This looks like an error we introduced in version 0.3.9 and which was fixed in #111 (but there wasn't a new release since).

Can you please check if the `master` branch works for you?

**Comment by mgeier** [comment]
@bitworm I've just made a new release 0.3.10, can you please try it?
